config/vufind/RateLimiter.yaml

```diff
@@ -109,9 +110,10 @@ Storage:
 # turnstileRateLimiterSettings Defined like rateLimiterSettings, but when the limit is
 #                              reached, a Turnstile challenge is used instead of simply
 #                              returning a 429. The challenge result is cached. 
+#                              To show the challenge immediately, use policy reject_all.
 #                              Failing the challenge results in standard 429 behavior.
-#                              Passing the challenge bypasses this check, but note the
-#                              rateLimiterSettings quota is still separately applied.
+#                              Passing the challenge bypasses this Turnstile check, but
+#                              note the rateLimiterSettings quota is still separately applied.
```

Comment:
@EreMaijala does this help at all with your concern?

> One more question: The comments in Turnstile config include this:

```
Passing the challenge bypasses this check, but note the
rateLimiterSettings quota is still separately applied.
```
> I suppose this could be clarified a bit since to me this sounds like RejectAll would be considered after passing the challenge. :)

I think it's clearer now explaining how reject_all affects things before talking about the consequences for success or failure.